lib/tumblr.js

```diff
@@ -37,6 +41,22 @@ var calls = {
     };
   },
 
+  postEditing: function(blogName, options, callback) { 
+    options = setDefaultPostOptions(options);
+    return function(blogName, options, callback) { 
+      this._post(blogPath(blogName, '/post/edit'), options, callback);
+    };
+  },
+
+  setDefaultPostOptions: function(options) {
```

Comment:
i think you want to make this its own function outside of the object because you're trying to use `setDefaultPostOptions()`, not `calls.setDefaultPostOptions()`